docs/tutorials/vis_cpu_tutorial.ipynb

```diff
@@ -0,0 +1,368 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # ra, dec in one array (rad)
</pre><p>Why not just construct <code>ra</code>  and <code>dec</code>  in radians to begin with?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HERA-Team/vis_cpu/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
I have modified the notebook as suggested.